### ISSUE TITLE
NameError when loaded in another engine

### ISSUE THREAD

**Issue by treenewbee** [galetahub/ckeditor#565]
In mongoid orm, If ckeditor is loaded in another engine, and respond_to?(:current_user) the following error will occur when you create an asset: 

NameError (uninitialized constant Assetable):
  ckeditor (4.1.1) lib/ckeditor/helpers/controllers.rb:17:in `ckeditor_before_create_asset'
  ckeditor (4.1.1) app/controllers/ckeditor/application_controller.rb:15:in`respond_with_asset'
  ckeditor (4.1.1) app/controllers/ckeditor/pictures_controller.rb:12:in `create'

class_name need to be specified somewhere?

**Comment by kyledecot** [comment]
@treenewbee Were you able to figure out a fix for this? If so could you do a pull request?

**Comment by treenewbee** [comment]
I took a closer look at it. I think I found where problem is:

My user is using active_record, but asset is using mongoid

ckeditor/lib/ckeditor/orm/mongoid.rb
line 34
belongs_to :assetable, :polymorphic => true

I guess the relation will not work until mongoid can work with active_model. My thought is there should be a config to toggle assetable relation or at least make it pass ckeditor_before_create_asset. Let me know if you want me to do pull request on this.
